src/main/java/com/paneedah/mwc/items/guns/SupernovaFactory.java

```diff
@@ -45,7 +45,7 @@ public Item createGun(CommonProxy commonProxy) {
         .withCrosshairRunning("Running")    
         .withShellCasingEjectEnabled(false)
         .withCrosshairZoomed("Sight")
-        .withInaccuracy(15)
+        .withInaccuracy(13)
```

Comment:
please revert this change (I will do extended research on balance of weapons at a later date)